odoo/openupgrade/doc/source/migration_details.rst

```diff
@@ -6,27 +6,65 @@ and upgrade your database by calling odoo-bin (or openerp-server) directly. You
 will want to do this when you are working on developing migration scripts for
 uncovered modules.
 
+General Tips
+++++++++++++
+
+* If you migrate your database through multiple versions make sure you use separate directories. If you use one directory and just check out a new branch the remaining *.pyc* files will cause trouble.
+
+* When installing the openupgradelib make sure you check out the latest version from github to get the latest updates and fixes. Either by following the method from *migrate.py* or installing via pip like this::
+
+    $ pip install --upgrade git+git://github.com/OCA/openupgradelib.git
+
+Steps for manual migration
+++++++++++++++++++++++++++
+
 1. Check out the OpenUpgrade source code from Github for the branches you
    need. Each branch migrates to its version from the previous version, so
    branch 7.0 migrates from 6.1 to 7.0. If you are skipping versions, you still
    need to run all the branches in between.
 
+   .. code-block:: bash
+
+       $ git clone https://github.com/OCA/OpenUpgrade.git
+       $ cd OpenUpgrade
+       $ git checkout 11.0
+       $ pip install -r requirements.txt
+       $ pip install --upgrade git+git://github.com/OCA/openupgradelib.git
+       $ cd ..
+
 2. Compare your set of installed modules with the modules that are covered by
    the OpenUpgrade Addons branch you are using. Upgrading a database that has
    uncovered modules installed is likely to fail. Authoritative in this respect
    is the existence and contents of a *user_notes.txt* file in the
    migrations/[version] subdirectory of each module. We also try to indicate
    module coverage in the documentation but it sometimes lags behind.
 
+   .. code-block:: bash
+
+       $ diff odoo/addons OpenUpgrade/addons/ | grep -v "Common subdirectories" | grep -v "OpenUpgrade"
```

Comment:
What does this do?

Reply:
this compares the addons directory of odoo with OpenUpgrade and ignores lines with "Common subdirectory" (existing in OpenUpgrade) or "OpenUpgrade" (existing only in OpenUpgrade, hence not installed on the odoo server)

Reply:
Still not clear what you are trying to do here. I assume `OpenUpgrade` is the distribution of openupgrade that you cloned under `1` , but what is `odoo` in this case?

Reply:
I don't see this relevant for a migration